packages/web3-eth/types/index.d.ts

```diff
@@ -146,9 +146,16 @@ export class Eth {
     ): Promise<number>;
 
     getGasPrice(
+        blockCount: number,
+        newestBlock: string,
+        rewardPercentiles: number[],
         callback?: (error: Error, gasPrice: string) => void
     ): Promise<string>;
 
+    getFeeHistory(
+        callback?: (error: Error, feeHistory: FeeHistoryResult) => void
```

Comment:
This should contain `blockCount`, `newestBlock` and `rewardPercentiles`?

Reply:
I think you mixed up `getGasPrice` and `getFeeHistory`?

Reply:
Also `rewardPercentiles` should be  marked as optional

Reply:
Actually I think the docs here are misleading https://github.com/ChainSafe/web3.js/pull/4190/files#diff-b1b5b8a010cd61ad800e528887f49730db04064d59d3ee5ed14de0f49f2ba11cR740

Looks like you still have to pass an empty array so I'm not sure if web3 should just do that as the default if that param isn't set? Would be more user friendly than having to pass an empty array.

Reply:
@GregTheGreek Whoops, forgot to push, thank you

@corymsmith I think web3 defaulting an empty array would be nice for the user, but we can't have two optional arguments in typescript, so I'm not sure how we could do this

Reply:
True, I suppose the only way to do it would be to add a formatter that formats `null` into `[]`